File: bootstrap/src/test/java/net/dontdrinkandroot/wicket/bootstrap/AbstractWicketTest.java
package net.dontdrinkandroot.wicket.bootstrap;

import org.apache.wicket.util.tester.WicketTester;
import org.junit.Before;


public class AbstractWicketTest
{

	protected WicketTester tester;


	@Before
	public void setUp()
	{
		this.tester = new WicketTester(new TestApplication());
	}
}

File: bootstrap/src/test/kotlin/net/dontdrinkandroot/wicket/bootstrap/test/AbstractWicketTest.kt
package net.dontdrinkandroot.wicket.bootstrap.test

import org.apache.wicket.util.tester.WicketTester
import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.BeforeEach

open class AbstractWicketTest {

    protected lateinit var tester: WicketTester

    @BeforeEach
    fun setUp() {
        tester = WicketTester(TestApplication())
    }

    @AfterEach
    fun tearDown() {
        tester.destroy()
    }
}